recipes/mmcv/meta.yaml

```
@@ -0,0 +1,130 @@
+{% set version = "1.5.3" %}
+{% set number = 0 %}
+{% set proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+{% if cuda_compiler_version != "None" %}
+{% set number = number + 200 %}
+{% endif %}
+
+package:
+  name: mmcv-recipe
+  version: {{ version }}
+
+source:
+  url: https://github.com/open-mmlab/mmcv/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 650929f52f1d0b2bae1bdaaa874e1e6dfdfa15f40bab5c366330488a9949ddd0
+build:
+  number: {{ number }}
+  skip: true  # [win]
+
+outputs:
+  - name: mmcv-full
+    build:
+      string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+      string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None"]
+      detect_binary_files_with_prefix: false
+      run_exports:
+        - {{ pin_subpackage('mmcv-full', max_pin='x.x') }}
+      missing_dso_whitelist:
+        # conda-forge::pytorch provides these libs
+        - "$RPATH/libc10_cuda.so"
+        - "$RPATH/libtorch_python.so"
+        - "$RPATH/libtorch_cpu.so"
+        - "$RPATH/libc10.so"
+        - "$RPATH/libc10_cuda.dylib"
+        - "$RPATH/libtorch_python.dylib"
+        - "$RPATH/libtorch_cpu.dylib"
+        - "$RPATH/libc10.dylib"
+    script: build_mmcv.sh  # [not win]
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        # this adds matching cuda requirement to run deps using __cuda package
+        - {{ compiler('cuda') }}    # [cuda_compiler_version != "None"]
+        - ninja
+      host:
+        - python
+        - pip
+        - requests
+        - future
+        - six
+        - typing
+        - pkg-config  # [unix]
+        - typing_extensions
+        - pytorch =*={{ proc_type }}*
```

```suggestion
        - pytorch ={{ pytorch }}={{ proc_type }}*
```

I think you will want to keep this syntax here, so that it pick up the pin from your custom `conda_build_config.yaml` or the global one.